README.md

```diff
@@ -105,11 +113,11 @@ NEXTAUTH_URL=http://localhost:3000/api/auth
 ```
 
 For more information, you can read the documentation [here](https://next-auth.js.org/configuration/options).
+**Example .env file can be found [here](./sample.env). You can rename this to .env to get started**
```

Comment:
This is all that I added to the README. Perhaps it would be better at the top. Not sure